infrastructure/code/code_test.go

```diff
@@ -654,14 +654,15 @@ func TestUploadAnalyzeWithAutofix(t *testing.T) {
 	)
 
 	t.Run(
-		"should not run autofix after analysis when is enabled but have disallowed extension",
+		"should not provide autofix code action when autofix enabled but issue not fixable",
```

Comment:
do you already have a positive test that it's added when fixable?

Reply:
Yes, it's called `should run autofix after analysis when is enabled` in the same test suite 👌